package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
```

Comment:
I think Windows does a better job in making pixel-perfect images, and yes I blame it on Mac.